src/js/components/chat-input.jsx

```diff
@@ -70,7 +78,7 @@ export class ChatInputComponent extends Component {
     }
 
     componentDidMount() {
-        this.resizeInput();
+        setTimeout(() => this.resizeInput());
```

Comment:
this makes sure to execute  `this.resizeInput` after the current stack, so usually after the HTML was actually flushed to the DOM and the widget has actual height and width (needed to resize the input)